test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/TagBuilderTest.cs

```diff
@@ -169,5 +169,127 @@ public void ReadingInnerHtml_LeavesHasInnerHtmlFalse()
             Assert.False(tagBuilder.HasInnerHtml);
             Assert.NotNull(innerHtml);
         }
+
+        [Fact]
+        public void RenderStartTag_RendersExpectedStartTag()
+        {
+            // Arrange
+            var tagBuilder = new TagBuilder("p");
+            var tag = tagBuilder.RenderStartTag();
```

Comment:
This is the `// Act` and `HtmlContentUtilities.HtmlContentToString(tag)` will clean up the `// Assert` part.